docs/source/private_data_tutorial.rst

```diff
@@ -19,16 +19,19 @@ configuring and using private data with Fabric:
 #. :ref:`pd-build-json`
 #. :ref:`pd-read-write-private-data`
 #. :ref:`pd-install-define_cc`
+#. :ref:`pd-register-identities`
 #. :ref:`pd-store-private-data`
 #. :ref:`pd-query-authorized`
 #. :ref:`pd-query-unauthorized`
 #. :ref:`pd-purge`
+#. :ref:`pd-transfer-asset`
 #. :ref:`pd-indexes`
 #. :ref:`pd-ref-material`
 
-This tutorial will deploy the `marbles private data sample <https://github.com/hyperledger/fabric-samples/tree/{BRANCH}/chaincode/marbles02_private>`__
+This tutorial will deploy the `asset transfer private data sample <https://github.com/hyperledger/fabric-samples/tree/master/asset-transfer-private-data/chaincode-go>`__
 to the Fabric test network to demonstrate how to create, deploy, and use a collection of
-private data. You should have completed the task :doc:`install`.
+private data and the associated ``README`` file contains additional details.
+You should have completed the task :doc:`install`.
 
```

Comment:
On second thought, it is a long intimidating set of steps for a first reader. So building a high level usecase description, will give the reader a context.

How about this as a top level section before "Build a collection definition JSON file"?

Asset transfer private data sample usecase:
--------------------------------------------
This sample demonstrates the use of three private data collections, `assetCollection, Org1MSPPrivateCollection & Org2MSPPrivateCollection` to transfer an asset between Org1 and Org2, using following usecase:

A member of Org1 creates a new asset, henceforth referred as owner. The public details of the asset, 
including the owner, are stored in the `assetCollection` collection. The asset is also created with an appraised 
value supplied by the owner. The appraised value is used by each participant to agree to the transfer of the asset, and is only stored in owner organization's collection. In our case, the initial appraisal value agreed by the owner is stored in the `Org1MSPPrivateCollection`.

To purchase the asset, the buyer needs to agree to the same appraised value as the asset owner. In this step, the buyer (a member of Org2) creates an agreement to trade and agree to an appraisal value using smart contract function `AgreeToTransfer`. This value is stored in `Org2MSPPrivateCollection` collection.

Now, the asset owner can transfer the asset to the buyer using smart contract function `TransferAsset`, which checks for the conditions that must be met for transfer to succeed.

Reply:
I would agree with this comment. I would put some the transaction flow summary at the top, to make the rest of the flow easier.

Reply:
Good add, done.